### DISCUSSION TITLE
adjust the beacon time, is it possible

### DISCUSSION THREAD

**Discussion by F4VSJ** [lora-aprs/LoRa_APRS_iGate#215] [Q&A] [answered]
Hello,
I'm using an Igate and lora tracker with great pleasure
I'm looking around in all .cpp files to find out where to shorten the beacon timer 
It is adjusted to 15 min and I want to have it all 10 min
Can somebody give me a hint?
Thanks and 73 de hans
F4VSJ

**Comment by wolfpcgn** [comment, marked as answer]
maybe "project_configuration.cpp" line 47 or "project-configuration.h" line 51
